apps/ejabberd/src/mod_mam_muc_odbc_arch.erl

```diff
@@ -662,3 +650,78 @@ maybe_encode_compact_uuid(undefined, _) ->
     undefined;
 maybe_encode_compact_uuid(Microseconds, NodeID) ->
     encode_compact_uuid(Microseconds, NodeID).
+
+
+%% ----------------------------------------------------------------------
+%% Optimizations
+
+packet_to_stored_binary(Packet) ->
+    case odbc_message_format() of
```

Comment:
The same comment applies to all other places where there is similar case like this one.